ChatBot/bot.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import logging
+
+from policy import CustomPolicy
+from rasa_core import utils
+from rasa_core.actions import Action
+from rasa_core.agent import Agent
+from rasa_core.channels.console import ConsoleInputChannel
+from rasa_core.events import SlotSet
+from rasa_core.interpreter import RasaNLUInterpreter
+from rasa_core.policies.memoization import MemoizationPolicy
+from rasa_core.actions.forms import (
+    EntityFormField,
+    FormAction
+)
+
+logger = logging.getLogger(__name__)
+
+
+class ActionPatientInformation(FormAction):
+    RANDOMIZE = False
+
+    @staticmethod
+    def required_fields():
+        return [
+            EntityFormField("gender", "gender"),
+            EntityFormField("age", "age"),
+        ]
+
+    @classmethod
+    def name(self):
+        return 'action_patient_information'
+
+    @classmethod
+    def submit(self, dispatcher, tracker, domain):
+        return [SlotSet("matches", "text")]
+
+
+class ActionSuggestNormal(Action):
+    @classmethod
+    def name(self):
+        return 'action_suggest_normal'
+
+    @classmethod
+    def run(self, dispatcher, tracker, domain):
+        medicines = ['Atovaquone', 'Chloroquine', 'Doxycycline', 'Mefloquine', 'Primaquine']
+        age = int(tracker.get_slot("age"))
+        if age < 8:
```

Comment:
can't we use a similar logic to check if age/gender already exists and not ask him again.

That should solve Issue @Buddhiprabha was referring to in the last meeting.

Reply:
Yes. If that is to be implemented, it would be done using these Python functions and conditionals.

Reply:
yes, I think we have another Issue open to track that.